source/Octopus.Cli/Exporters/FileSystemExporter.cs

```diff
@@ -20,10 +20,7 @@ public FileSystemExporter(IOctopusFileSystem fileSystem, ILogger log)
 
         public void Export<T>(string filePath, ExportMetadata metadata, T exportObject)
         {
-            var x = exportObject.ToDynamic(metadata);
```

Comment:
As `JsonConvert` and `JsonSerialization.GetDefaultSerializerSettings()` weren't available after we turned off the InternalsVisibleTo, a wrapper class was introduced to get access to the correct serialization settings.